libsolidity/parsing/Parser.cpp

```diff
@@ -226,16 +226,14 @@ ASTPointer<ImportDirective> Parser::parseImportDirective()
 	expectToken(Token::Import);
 	ASTPointer<ASTString> path;
 	ASTPointer<ASTString> unitAlias = make_shared<string>();
+	SourceLocation unitAliasLocation = currentLocation();
```

Comment:
Shouldn't this get the default value instead?

Reply:
yes.